tools/kvmexit.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
+#
+# kvmexit.py
+#
+# Display the exit_reason and its statistics of each kvm_exit
+# for all vcpus of all virtual machines. For example,
+# $./kvmexit.py
+#  TGID     PID      KVM_EXIT_REASON                     STAT
+#  1273551  1273568  EXIT_REASON_MSR_WRITE               6
+#  1274253  1274261  EXIT_REASON_EXTERNAL_INTERRUPT      1
+#  1274253  1274261  EXIT_REASON_HLT                     12
+#  ...
+#
+# @TGID: each vitual machine's pid in the user space.
+# @PID: the user space's thread of each vcpu of that virtual machine.
```

Comment:
Using "TID". Also, do you think TID is useful? I am not quite sure about it.

Reply:
> Using "TID". Also, do you think TID is useful? I am not quite sure about it.

Thanks. Yes, in some cases only one specific TID exits frequently or has an abnormal KVM_EXIT_REASON.